package.json

```diff
@@ -381,7 +381,6 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/register": "^7.5.5",
-    "@ethersproject/bignumber": "^5.7.0",
     "@lavamoat/allow-scripts": "^2.3.1",
     "@lavamoat/lavapack": "^5.2.4",
     "@metamask/auto-changelog": "^2.1.0",
```

Comment:
I think creating a comment thread for this will allow the conversation to continue after I merge 

@abdelrahmansheta16 does your cursor change to a pointer hand when you hover over the row?